media-sound/mp3gain/mp3gain-1.5.2.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Lossless mp3 normalizer with statistical analysis"
+DESCRIPTION="MP3Gain analyzes and adjusts mp3 files so that they have the same \
+volume. MP3Gain does not just do peak normalization, as many normalizers do. \
+Instead, it does some statistical analysis (using the replaygain algorithm) to \
+determine how loud the file actually sounds to the human ear.
+MP3Gain can adjust the volume in a completely lossless way by modifying the \
+mp3 file directly, without decoding and re-encoding. This works with all mp3 \
+players, i.e. no support for a special tag is required."
+HOMEPAGE="http://mp3gain.sourceforge.net/"
+COPYRIGHT="1995, 1996, 1997 Michael Hipp
```

Comment:
Thanks for the update!
May I ask you one last tiny edit?
```diff
-COPYRIGHT="1995, 1996, 1997 Michael Hipp
+COPYRIGHT="1995-1997 Michael Hipp
```
(Here this is not very important because there are only 3 years in the list, but since we try do that for all our recipes, it would be more consistent with the other recipes.)
Thanks!

Reply:
Shall we also drop the debugInfoPackage to please @korli, @pulkomandy and @mmlr ?
(This will allow to get green lights for the merge ;)

Reply:
It seems we can:
```diff
-defineDebugInfoPackage mp3gain \
-	$binDir/mp3gain
+extractDebugInfoPackage mp3gain
```